src/package.lisp

```diff
@@ -3,7 +3,7 @@
   (:use :cl)
   (:nicknames :crypto)
   (:shadow null)
-  (:import-from #:nibbles 
+  (:import-from #:nibbles
```

Comment:
My editor automatically removes trailing whitespace. Sorry if you consider this clutter in the PR.